templates/sensu.erb

```diff
@@ -1,21 +1,4 @@
-EMBEDDED_RUBY=<%= @use_embedded_ruby %>
-LOG_LEVEL=<%= @log_level %>
-LOG_DIR=<%= @log_dir %>
-<% if not @rubyopt.nil? -%>
-RUBYOPT="<%= @rubyopt %>"
-<% end -%>
-<% if not @gem_path.nil? -%>
-GEM_PATH="<%= @gem_path %>"
-<% end -%>
-<% if @deregister_on_stop -%>
-CLIENT_DEREGISTER_ON_STOP=true
-CLIENT_DEREGISTER_HANDLER="<%= @deregister_handler %>"
-<% end -%>
-SERVICE_MAX_WAIT="<%= @init_stop_max_wait %>"
-PATH=<%= @path %>
-<% if @confd_dir -%>
-CONFD_DIR="<%= @conf_dir %>,<%= Array(@confd_dir).join(",") %>"
-<% end -%>
-<% if @heap_size -%>
-HEAP_SIZE="<%= @heap_size %>"
+# File managed by Puppet
```

Comment:
We may also mantain all the if conditions in the template, it would just require more maintenance, as contante would need to be changed whenever a new parameter is added